packages/chain-adapters/src/evm/avalanche/AvalancheChainAdapter.ts

```diff
@@ -61,23 +60,21 @@ export class ChainAdapter extends EvmBaseAdapter<KnownChainIds.AvalancheMainnet>
   }
 
   async getGasFeeData(): Promise<GasFeeDataEstimate> {
-    const { gasPrice, fast, average, slow } = await this.api.getGasFees()
-
-    const scalars = { fast: bn(1.2), average: bn(1), slow: bn(0.8) }
+    const { fast, average, slow } = await this.api.getGasFees()
 
     return {
       fast: {
-        gasPrice: calcFee(gasPrice, 'fast', scalars),
+        gasPrice: fast.maxFeePerGas ?? '0',
```

Comment:
will be improved with the unchained legacy fee update coming up next, but there should always be `maxFeePerGas` available on chains that support eip1559 (ethereum, avalanche, polygon) even though it is typed as optional